src/plugins/dashboard/public/application/listing/dashboard_listing.js

```diff
@@ -163,6 +164,7 @@ export class DashboardListing extends React.Component {
   }
 
   getTableColumns() {
+    const [, dateFormat] = this.props.core.uiSettings.get('dateFormat:scaled')[3];
```

Comment:
This seems problematic since this setting can be edited and it's not guaranteed that there will always be 4 elements in this array. Can we just use `dateFormat`?

Reply:
Hey @tmarkley, thanks for the comments.
This change was done, after receiving a previous CR comment to use "advanced settings format".
Regardless, I understand what you wrote about the number of elements.

Still, I'm not sure I understood the offered solution, 
The dateFormat you see here only uses us as the format itself. i.e we do moment(updatedAt).format(dateFormat).

Do you want me to write "dateFormat " as a static value without accessing to the "advanced settings format"? i.e
 moment(updatedAt).format('["PT1H", "YYYY-MM-DD HH:mm"]')?

If not the case, will appreciate an alternative code solution from you if possible. Thanks

Reply:
Hi @RoyiSitbon, apologies for the confusion. I was referring to the `dateFormat` advanced setting.

Reply:
Hey @tmarkley. Using dateFormat it will look like that:
<img width="166" alt="Screen Shot 2022-05-13 at 9 29 47" src="https://user-images.githubusercontent.com/12782945/168224463-1a416f0d-bc4c-4458-8a75-d8f5d521e021.png">

Is it fine with you? Thanks

Reply:
I'm ok with this, since this will be consistent with date time formats anywhere else in the system. If the user changes their preferences, then this will also change.

Reply:
Great, changed :)